src/components/Toggle/tests/Toggle.test.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
+import {noop} from '@shopify/javascript-utilities/other';
+import {mountWithAppProvider, findByTestID} from 'test-utilities';
+import Labelled from '../../Labelled';
+import TextContainer from '../../TextContainer';
+import Icon from '../../Icon';
+import Toggle from '../Toggle';
+
+describe('<Toggle />', () => {
+  const mockProps = {
+    label: 'label',
+    onChange: noop,
+  };
+
+  describe('role', () => {
+    it('gets set to checkbox', () => {
+      const toggle = mountWithAppProvider(<Toggle {...mockProps} />);
+      const input = findByTestID(toggle, 'ToggleInput');
+
+      expect(input.prop('role')).toBe('checkbox');
+    });
+  });
+
+  describe('checked', () => {
+    it('sets aria-checked if true', () => {
+      const toggle = mountWithAppProvider(<Toggle {...mockProps} checked />);
+      const input = findByTestID(toggle, 'ToggleInput');
+
+      expect(input.prop('aria-checked')).toBeTruthy();
```

Comment:
We should avoid `toBeTruthy` and instead be literal in our expectations. In this case, the test can give a false positive because `expect('false').toBeTruthy()` will pass.

Reply:
Although, after looking at the Enzyme API it looks like the `prop` method does some magic to turn these into booleans.

Reply:
```suggestion
      expect(input.prop('aria-checked')).toBe(true);
```